src/Uno.UWP/Devices/Bluetooth/BluetoothLEDevice.cs

```diff
@@ -13,6 +15,13 @@ public partial class BluetoothLEDevice : global::System.IDisposable
 		public string Name { get; internal set; }
 		public BluetoothLEAppearance Appearance { get; internal set; }
 
+		private BluetoothLEDevice(string name, BluetoothLEAppearance appearance)
+		{
+			// dummy for Error CS8618  Non-nullable property 'Value' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.
+			Name = name;
+			Appearance = appearance;
```

Comment:
Instead, let's set default values that make sense, since we don't populate the values properly yet. Name could be "N/A" and appearance could be a new instance of `BluetoothLEAppearance` which also returns default values.

Otherwise, developers using these properties may fail with NREs assuming it cannot be because of the nullable annotations.

Reply:
Can I "resolve conversation"?